cli/core/docs/cli/plugin_implementation_guide.md

```diff
@@ -5,7 +5,7 @@
 The Tanzu CLI was built to be extensible across teams and be cohesive across SKUs. To this end, the Tanzu CLI provides
```

Comment:
Also need to fix links at lines: 
- 42
- 162
-

Reply:
Fixed.